ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js

```diff
@@ -209,6 +213,11 @@
         'Acrid Stream': 'Ätzende Strömung',
         '2,000-mina swipe': '2000 Mina-Hiebe',
         '2,000-mina swing': '2000 Mina-Schwung',
+        'Swing/Swipe/Cyclone': '',
```

Comment:
?

Reply:
seams i missed this one after translating the rest...